keyboards/chew/split/keyboard.json

```
@@ -33,18 +21,12 @@
             }
         },
         "serial": {
-            "driver": "vendor",
-            "pin": "GP11"
+            "driver": "vendor"
```

This should not be reverted
```suggestion
            "driver": "vendor",
            "pin": "GP11"
```